sass/app.scss

```diff
@@ -6,3 +6,7 @@
 @import 'core/all';
 @import 'components/all';
 @import 'util/all';
+
+* {
```

Comment:
@brendansudol This seemed to fix the problem of breaking the navigation and other layout on the page for me